spotbugs/src/main/java/edu/umd/cs/findbugs/ClassAnnotation.java

```diff
@@ -198,6 +199,11 @@ public void writeXML(XMLOutput xmlOutput, boolean addMessages, boolean isPrimary
             attributeList.addAttribute("primary", "true");
         }
 
+        if (!getJavaAnnotationNames().isEmpty()) {
+            attributeList.addAttribute("classjas", // similar to classname, short for classJavaAnnotations
```

Comment:
@phisad is 'classjas' just made up naming?  Seems odd even from change log.  Why not just 'classAnnotations'?  We already know its java, that is redundant.  At least then it follows the idea you have there and is more clear on name without new jargon.  Then you probably don't need to describe why its named that way...

Reply:
Thanks for the comment! I was also thinking about "classAnnotations" but felt that it might be confused with the ClassAnnotation from spotbugs. But I think you are right, that a user of the XML file will not know that the term "classAnnotation" is also otherwise used. If you say that this is clearer, then I will certainly adjust it.

Reply:
Good point, thanks for calling that out, that certainly would cause unnecessary confusion. how about 'classAnnotationNames' which is more in line with what its loading there?

Reply:
OK, I changed the attribute in the XMLs from _classjas_ to _classAnnotationNames_